### DISCUSSION TITLE
在启动器中添加mod

### DISCUSSION THREAD

**Discussion by somedude000** [Meloong-Git/PCL#3872] [提问] [answered]
新版本pcl提示可以直接把mod拖到pcl窗口以添加mod，但是我试了把java文件拖到窗口后无事发生，我想问下这是我操作有问题还是bug

**Comment by allMagicNB** [comment, marked as answer]
如果你说的是模组 Jar 文件的话，请检查下你有没有以管理员身份运行 PCL……

**Reply by Silverteal** [reply]
虽然您是高贵的协作者，但鄙人仍然建议您略微降低一点口气以免可能的误解造成的不愉快……

**Comment by MoYuan-CN** [comment]
你把什么东西放入了 PCL 窗口？

**Comment by WForst-Breeze** [comment]
您想说的是 jar 文件吧？看看是否使用了管理员身份运行 PCL？如若没有可以上传一下日志看看

**Comment by kong-que-dnf123** [comment]
要不试试不用管理员运行...?

**Comment by somedude000** [comment]
感谢各位，我试了不以管理员运行之后就可以了
